openhtf/io/output/print_ToConsole.py

```diff
@@ -0,0 +1,89 @@
+"""Module to display test summary on console."""
+import os
+import sys
+from openhtf import util
+from openhtf.io import test_record
+from openhtf.util import measurements as mea
+import datetime
```

Comment:
Imports should be in three sections: (1) built-ins, (2) third-party deps, (3) local to this codebase. See the style guide [here.](https://www.python.org/dev/peps/pep-0008/#imports) So this should be:

```
import datetime
import os
import sys

from openhtf import util
from openhtf.io import test_record
from openhtf.util import measurements
```